examples/gno.land/r/gnoland/hof/rendering.gno

```diff
@@ -0,0 +1,121 @@
+package hof
+
+import (
+	"gno.land/p/demo/fqname"
+	"gno.land/p/demo/seqid"
+	"gno.land/p/demo/ufmt"
+)
+
+const (
+	MaxWidgetSize = 5
+	likesBar      = "#### [%d 👍](/r/gnoland/hof?help&__func=Upvote&exhibID=%s&itemID=%s) - [%d 👎](/r/gnoland/hof?help&__func=Downvote&exhibID=%s&itemID=%s)\n\n"
+	delBar        = "[Delete](/r/gnoland/hof?help&__func=Delete&exhibID=%s&itemID=%s)\n\n"
+)
+
+func Render(path string) string {
+	out := "# Hall of Fame\n\n"
+
+	// Render permanent exhib
+	permanent, _ := exhibitions.Get(seqid.ID(0).String())
+	out += permanent.(*Exhibition).Render(path == "dashboard")
+
+	if exhibitions.Size() > 1 {
+		out += "---\n\n"
+	}
+
+	exhibitions.Iterate(seqid.ID(1).String(), "", func(key string, value interface{}) bool {
+		e := value.(*Exhibition)
+		out += e.Render(path == "dashboard")
+
+		out += "---\n\n"
+		return false
+	})
+
+	return out
+}
+
+func (e Exhibition) Render(dashboard bool) string {
+	out := ufmt.Sprintf("## %s\n\n", e.title)
+	out += ufmt.Sprintf("%s\n\n", e.description)
+	out += ufmt.Sprintf("Exhibition ID: %s\n\n", e.id.String())
+
+	if e.items.Size() == 0 {
+		out += "No items in this exhibition currently.\n\n"
+		return out
+	}
+
+	out += "<div class='columns-2'>\n\n"
+
+	e.items.ReverseIterate("", "", func(key string, value interface{}) bool {
```

Comment:
should use a pager :)

Reply:
Let's add it after we merge your avl/pager PR